index.html

```diff
@@ -135,90 +135,75 @@
+            </p>
+
+            <p class="note">
+                There is no requirement that the publisher publishes all these states; it may choose to publish only one. I.e., there can be situations where the <a>PWP</a> is only published unpacked (so only the locator of the published unpacked <a>PWP</a> exists), or only published packed (so only that locator exists).
+            </p>
+        </li>
+
+        <li>A <dfn data-lt="manifest|manifest file" >manifest (file)</dfn> is a special resource containing fundamental information about a PWP. The serialization syntax and the content of a manifest is not defined in this document. A media type (see [[rfc6838]]) is also associated with a manifest (or a particular serialization thereof), that can be used when a particular instance is retrieved through standard HTTP(S) protocol.</li>
+
+        <li>A <dfn data-lt="manifest item|manifest items">manifest item</dfn> is an information item within the <a>manifest file</a> that can be retrieved by, e.g., a <a>PWP Processor</a>. These may include metadata, information on rendering order and rendition, etc. For the purpose of this document it is important that <strong>L<sub>u</sub></strong> and <strong>L<sub>p</sub></strong> are such manifest items, i.e., they can be part of a <a>manifest</a>.</li>
+
+        <li>A <dfn>PWP manifest</dfn> or <dfn>complete manifest</dfn> is a <a>manifest</a> that contains <em>all</em> <a>manifest items</a> that are required by a <a>PWP processor</a>. The complete list of required items are not defined here; for the purpose of this document it suffices to state that <strong>L</strong>, <strong>L<sub>u</sub></strong>, and <strong>L<sub>p</sub></strong> MUST be part of a <a>PWP manifest</a> if those <a data-lt="state">states</a> are indeed available on the <a>server</a>.</li>
+
+        <li>The <dfn>combination of manifests</dfn>, denoted by the ⊕ operation (as in <strong>M</strong>&nbsp;=&nbsp; <strong>M<sub>1</sub></strong>&nbsp;⊕&nbsp;<strong>M<sub>2</sub></strong>) is the creation of a new <a>manifest</a>, consisting of <a data-lt="manifest item">manifest items</a> originating from <strong>M<sub>1</sub></strong> and <strong>M<sub>2</sub></strong>.
+
+            <p>The precise definition of a <a>manifest</a> should also specify how different <a>manifest items</a> are combined during this operation. For the purpose of this document it is only required, that:</p>
+            <ul>
+                <li>the value of <strong>L</strong> MUST be identical, if present, in both <strong>M<sub>1</sub></strong> and <strong>M<sub>2</sub></strong>; and</li>
+
+                <li>the value of <strong>L<sub>u</sub></strong> (resp. <strong>L<sub>p</sub></strong>) in <strong>M<sub>2</sub></strong> MUST have a higher priority. I.e., if present in <strong>M<sub>2</sub></strong>, this is the value added to <strong>M</strong>, regardless of whether a similar value is also specified in <strong>M<sub>2</sub></strong> or not.</li>
```

Comment:
Oops, I am sorry, you are absolutely right! Just changed it on the repo




> On 11 Mar 2016, at 11:51, Ben De Meester <notifications@github.com> wrote:
> 
> In index.html <https://github.com/w3c/dpub-pwp-loc/pull/18#discussion_r55815660>:
> 
> > +            </p>
> > +        </li>
> > +
> > +        <li>A <dfn data-lt="manifest|manifest file" >manifest (file)</dfn> is a special resource containing fundamental information about a PWP. The serialization syntax and the content of a manifest is not defined in this document. A media type (see [[rfc6838]]) is also associated with a manifest (or a particular serialization thereof), that can be used when a particular instance is retrieved through standard HTTP(S) protocol.</li>
> > +
> > +        <li>A <dfn data-lt="manifest item|manifest items">manifest item</dfn> is an information item within the <a>manifest file</a> that can be retrieved by, e.g., a <a>PWP Processor</a>. These may include metadata, information on rendering order and rendition, etc. For the purpose of this document it is important that <strong>L<sub>u</sub></strong> and <strong>L<sub>p</sub></strong> are such manifest items, i.e., they can be part of a <a>manifest</a>.</li>
> > +
> > +        <li>A <dfn>PWP manifest</dfn> or <dfn>complete manifest</dfn> is a <a>manifest</a> that contains <em>all</em> <a>manifest items</a> that are required by a <a>PWP processor</a>. The complete list of required items are not defined here; for the purpose of this document it suffices to state that <strong>L</strong>, <strong>L<sub>u</sub></strong>, and <strong>L<sub>p</sub></strong> MUST be part of a <a>PWP manifest</a> if those <a data-lt="state">states</a> are indeed available on the <a>server</a>.</li>
> > +
> > +        <li>The <dfn>combination of manifests</dfn>, denoted by the ⊕ operation (as in <strong>M</strong>&nbsp;=&nbsp; <strong>M<sub>1</sub></strong>&nbsp;⊕&nbsp;<strong>M<sub>2</sub></strong>) is the creation of a new <a>manifest</a>, consisting of <a data-lt="manifest item">manifest items</a> originating from <strong>M<sub>1</sub></strong> and <strong>M<sub>2</sub></strong>.
> > +
> > +            <p>The precise definition of a <a>manifest</a> should also specify how different <a>manifest items</a> are combined during this operation. For the purpose of this document it is only required, that:</p>
> > +            <ul>
> > +                <li>the value of <strong>L</strong> MUST be identical, if present, in both <strong>M<sub>1</sub></strong> and <strong>M<sub>2</sub></strong>; and</li>
> > +
> > +                <li>the value of <strong>L<sub>u</sub></strong> (resp. <strong>L<sub>p</sub></strong>) in <strong>M<sub>2</sub></strong> MUST have a higher priority. I.e., if present in <strong>M<sub>2</sub></strong>, this is the value added to <strong>M</strong>, regardless of whether a similar value is also specified in <strong>M<sub>2</sub></strong> or not.</li>
> Shouldn't the very last M2 actually be M1?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/w3c/dpub-pwp-loc/pull/18/files#r55815660>.
> 


----
Ivan Herman, W3C
Digital Publishing Lead
Home: http://www.w3.org/People/Ivan/
mobile: +31-641044153
ORCID ID: http://orcid.org/0000-0003-0782-2704